src/irmin-pack/IO.ml

```diff
@@ -60,37 +60,38 @@ module Unix : S = struct
 
     let v fd = { fd; cursor = 0L }
```

Comment:
I think we can remove the `cursor`, it was used to prevent unnecessary calls to `lseek`.

Reply:
Indeed. This `cursor` actually still exists in Index, even though there's no `lseek` being done there... Will remove from both.